website/views/shared/tasks/edit/repeatables.jade

```diff
@@ -1,34 +1,34 @@
-fieldset.option-group.advanced-option(ng-show="task.type === 'daily'")
-    legend.option-title
-      span.hint(popover-title=env.t('startDateHelpTitle'), popover=env.t("startDateHelp"), popover-trigger='mouseenter')
-        =env.t('startDate')
-    input.form-control(type='text', ng-model='task._edit.startDate',
-      datepicker-popup='{{::user.preferences.dateFormat}}', is-open='datepickerOpened',
-      ng-click='datepickerOpened = true', ng-disabled='!canEdit(task)')
-    
-    br
-
-    //- select.form-control(ng-model='task._edit.frequency', ng-disabled='!canEdit(task)')
-    //-   option(value='daily')=env.t('daily')
-    //-   option(value='weekly')=env.t('weekly')
-    //-   option(value='monthly')=env.t('monthly')
-    //-   option(value='yearly')=env.t('yearly')
-
-    select.form-control(ng-model='task._edit.frequency', ng-disabled='!canEdit(task)')
-      option(value='weekly')=env.t('repeatWeek')
-      option(value='daily')=env.t('repeatDays')
-
-    include ./dailies/repeat_options
-
-    .form-group(ng-show='task._edit.frequency === "monthly"')
-      legend.option-title=env.t('repeatsOn')
-      label
-        input(type="radio", ng-model='task._edit.repeatsOn', value='dayOfMonth')
-        =env.t('dayOfMonth')
-      label
-        input(type="radio", ng-model='task._edit.repeatsOn', value='dayOfWeek')
-        =env.t('dayOfWeek')
-
-    //- .form-group
-    //-   legend.option-title=env.t('summary')
+fieldset.option-group.advanced-option(ng-show="task.type === 'daily'")
```

Comment:
Append `?w=1` to the URL to ignore whitespace changes, and you'll see that only whitespace changed.